android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -314,6 +315,24 @@ public boolean hasUserConsent()
         return AppLovinPrivacySettings.hasUserConsent( maybeGetCurrentActivity() );
     }
 
+    @ReactMethod()
+    public void showConsentDialog(final Callback callback)
+    {
+        Activity currentActivity = maybeGetCurrentActivity();
+        if ( currentActivity == null ) return;
+
+        if ( sdk == null ) return;
+
+        sdk.getUserService().showConsentDialog( currentActivity, new AppLovinUserService.OnConsentDialogDismissListener()
```

Comment:
Let's just copy what we have in other plugins for consistency, i.e.

https://github.com/AppLovin/Unity-Plugin/blob/master/Android/Android-Unity-Plugin/src/main/java/com/applovin/mediation/unity/MaxUnityPlugin.java#L334-L340

Reply:
Ok, I'll copy it from the unity.